CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ### New features
 
+#### We've adjusted our responsive type scale
```

Comment:
If you want to change this to an action for the user it could be something like:
'Assess and adjust your service to the new typography scale'